### DISCUSSION TITLE
Add/Validate Recaptcha In my Customized Login

### DISCUSSION THREAD

**Discussion by YoumnaJ** [OrchardCMS/OrchardCore#15163] [Q&A] [answered]
Hi , I add a Recaptcha to my custom Login page but I need a way to validate Recaptcha After clicking Login.
Adding Recaptcha in this way 
![2024-01-25_8-21-06](https://github.com/OrchardCMS/OrchardCore/assets/154245181/5472b5d7-1a50-4063-8347-0cfba59beda8)

**Comment by hishamco** [comment, marked as answer]
You can use `ReCaptchaService.ValidateCaptchaAsync()`
